database-plugins/src/main/java/io/cdap/plugin/DBRecord.java

```diff
@@ -262,14 +270,20 @@ private void writeToDB(PreparedStatement stmt, Schema.Field field, int fieldInde
       switch (fieldLogicalType) {
         case DATE:
           stmt.setDate(sqlIndex, Date.valueOf(record.getDate(fieldName)));
+          bytesWritten += Long.BYTES;
```

Comment:
Is it possible for this `writeToDB` method to be called multiple times for the same DBRecord? If so the count would be incorrect right?

Reply:
`writeToDB` is called once per DBRecord. Call stack looks like ETLDBOutputFormat.DBRecordWriter.write -> DBRecord.write -> DBRecord.writeToDB